content/en/docs/reference/glossary/cgroups.md

```diff
@@ -0,0 +1,13 @@
+---
+title: cgroup
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  cgroup (abbreviated from control group) is a Linux kernel feature that limits, accounts for, and isolates the resource usage (CPU, memory, disk I/O, network) for a collection of processes.
```

Comment:
```suggestion
  A group of Linux processes with optional resource isolation, accounting and limits.
```

@ljnaresh do you prefer this short description (which will be used for a tooltip)? If you prefer your original wording, that's OK by me.